### DISCUSSION TITLE
Structure order dependant on locale

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9380] [Ideas]
> Created by: Eric (ericchantigny@msn.com) on 2015/11/06 19:44:14 +0000
> Votes at time of UserVoice import: 4

-----------------------------------

I usually use the Structures to put things in a certain order, which I love about Craft.  Then, in the front-end, when I call the entries of the structure, they come in the order I’ve set, no matter what that is.

Unfortunately, that is not possible for different locales.  Sure, we can programmatically ask for a certain order (alpha, date, …), but it can’t be made manually in the CP, specific to a language.  Maybe this could be a toggle in the Sections types?

Thanks

**Comment by brettburwell** [comment]
Any chance this is something that might be handled by Craft 3's improved localization?

**Comment by brandonkelly** [comment]
@brettburwell No, sorry.

**Comment by brettburwell** [comment]
@brandonkelly Good to know. Thanks for the quick reply!

**Comment by sa3dany** [comment]
I've ran into this recently when trying to sort some part of a structure in ascending order per site (different language). I think that if you can disable an entry per site (current behavior) you should be able to give an entry a certain order per site too.
